### PR TITLE
Marking repo as Deprecated and removing codeship steps

### PR DESCRIPTION
**Pull request by kebringer-cb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Zscaler ZIA Sandbox Connector for VMware Carbon Black Cloud
+# DEPRECATED: Zscaler ZIA Sandbox Connector for VMware Carbon Black Cloud
+
+This integration is deprecated and no longer maintained. Please contact support if you are interested in this integration.
```
Comment (avanbrunt-cb) [inline README.md:3]: Same comment as the dev network phrasing

File: codeship-steps.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
-- name: linters
-  service: testing
-  command: bin/linter.sh
-- name: tests
-  service: testing
-  command: bin/run_tests.sh
+# Removing codeship steps as this project is no longer maintained
```
Comment (avanbrunt-cb) [inline codeship-steps.yml:1]: We should delete the pipeline in codeship and remove this file

Comment (kebringer-cb) [reply codeship-steps.yml]: Done

File: README.md (hunk 1 above)
Comment (kebringer-cb) [reply README.md:3]: done

**Review by avanbrunt-cb — APPROVED** [review]

**Pull request merged**